stock_available_immediately/product.py

```diff
@@ -55,7 +53,7 @@ def _product_available(self, cr, uid, ids, field_names=None,
         if 'immediately_usable_qty' in field_names:
             for product_id, stock_qty in res.iteritems():
                 res[product_id]['immediately_usable_qty'] = \
-                    stock_qty['qty_available'] + stock_qty['outgoing_qty']
+                    stock_qty['qty_available'] - stock_qty['outgoing_qty']
```

Comment:
In 8.0 outgoing qty is calculated as an absolute quantity.
https://github.com/odoo/odoo/blob/8.0/addons/stock/product.py#L380-L381
https://github.com/odoo/odoo/blob/8.0/addons/stock/product.py#L112-L152

In 7.0 that wasn't the case amounts are subtracted, and returned as a negative value:
https://github.com/odoo/odoo/blob/7.0/addons/stock/product.py#L334-338

therefore our interpretation of immediately_usable_qty needed to be changed.